This change fixes the test suite on big endian ppc64 systems by using correct
word instructions on an `int *` rather than doubleword ones.

--- tests/dummy/patchentrypoints.c
+++ tests/dummy/patchentrypoints.c
@@ -176,9 +176,9 @@ static void patch_ppc64(char *writeEntry, const char *execEntry,
         // 1000:
         0x7D2903A6,     //  mtctr 9
         0xE96C0020,     //  ld    11, 9000f-1000b(12)
-        0xE92B0000,     //  ld    9, 0(11)
+        0x812B0000,     //  lwz   9, 0(11)
         0x39290001,     //  addi  9, 9, 1
-        0xF92B0000,     //  std   9, 0(11)
+        0x912B0000,     //  stw   9, 0(11)
         0x7D2902A6,     //  mfctr 9
         0x4E800020,     //  blr
         0x60000000,     //  nop
